### PR TITLE
Translate docs/concepts/storage/volume-snapshots/ into Japanese

### LINKED ISSUE #38116: [ja] Translate docs/concepts/storage/volume-snapshots/ into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate https://kubernetes.io/docs/concepts/storage/volume-snapshots/ into Japanese.
**Why is this needed**
This page has not been translated.
**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by KobayashiD27**
close #38116

### CODE REVIEW

File: content/ja/docs/concepts/storage/volume-snapshots.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+`VolumeSnapshotClass`を使用すると、`VolumeSnapshot`に属するさまざまな属性を指定できます。これらの属性は、ストレージシステム上の同じボリュームから取得されたスナップショット間で異なる場合があるため、`PersistentVolumeClaim`の同じ`StorageClass`を使用して表現することはできません。
+
+ボリュームスナップショットは、完全に新しいボリュームを作成することなく、特定の時点でボリュームの内容をコピーするための標準化された方法をKubernetesユーザーに提供します。この機能により、たとえばデータベース管理者は、編集または削除の変更を実行する前にデータベースをバックアップできます。
+
+この機能を使用する場合、ユーザーは次のことに注意する必要があります。
+
+- APIオブジェクト`VolumeSnapshot`、`VolumeSnapshotContent`、および`VolumeSnapshotClass`は{{< glossary_tooltip term_id="CustomResourceDefinition" text="CRD" >}}であり、コアAPIの一部ではありません。
+- `VolumeSnapshot`のサポートは、CSIドライバーでのみ利用できます。
+- `VolumeSnapshot`の展開プロセスの一環として、Kubernetesチームは、コントロールプレーンに展開されるスナップショットコントローラーと、CSIドライバーと共に展開されるcsi-snapshotterと呼ばれるサイドカーヘルパーコンテナーを提供します。`VolumeSnapshot`および`VolumeSnapshotContent`オブジェクトを管理し、`VolumeSnapshotContent`オブジェクトの作成と削除を担当します。サイドカーcsi-snapshotterは、`VolumeSnapshotContent`オブジェクトを監視し、CSIエンドポイントに対して`CreateSnapshot`および`DeleteSnapshot`操作をトリガーします。
```
Comment (nasa9084) [inline content/ja/docs/concepts/storage/volume-snapshots.md:29]:
```suggestion
- `VolumeSnapshot`の展開プロセスの一環として、Kubernetesチームは、コントロールプレーンに展開されるスナップショットコントローラーと、CSIドライバーと共に展開されるcsi-snapshotterと呼ばれるサイドカーヘルパーコンテナを提供します。`VolumeSnapshot`および`VolumeSnapshotContent`オブジェクトを管理し、`VolumeSnapshotContent`オブジェクトの作成と削除を担当します。サイドカーcsi-snapshotterは、`VolumeSnapshotContent`オブジェクトを監視し、CSIエンドポイントに対して`CreateSnapshot`および`DeleteSnapshot`操作をトリガーします。
```

**Comment by atoato88** [comment]
The pages on preview site are following:
- https://deploy-preview-38141--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/storage/volume-snapshots
- https://deploy-preview-38141--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/storage/persistent-volumes

**Review by atoato88 — COMMENTED** [review]
Thank you for create this PR!
I left some comments.

File: content/ja/docs/concepts/storage/volume-snapshots.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+`VolumeSnapshotClass`を使用すると、`VolumeSnapshot`に属するさまざまな属性を指定できます。これらの属性は、ストレージシステム上の同じボリュームから取得されたスナップショット間で異なる場合があるため、`PersistentVolumeClaim`の同じ`StorageClass`を使用して表現することはできません。
+
+ボリュームスナップショットは、完全に新しいボリュームを作成することなく、特定の時点でボリュームの内容をコピーするための標準化された方法をKubernetesユーザーに提供します。この機能により、たとえばデータベース管理者は、編集または削除の変更を実行する前にデータベースをバックアップできます。
+
+この機能を使用する場合、ユーザーは次のことに注意する必要があります。
+
+- APIオブジェクト`VolumeSnapshot`、`VolumeSnapshotContent`、および`VolumeSnapshotClass`は{{< glossary_tooltip term_id="CustomResourceDefinition" text="CRD" >}}であり、コアAPIの一部ではありません。
+- `VolumeSnapshot`のサポートは、CSIドライバーでのみ利用できます。
+- `VolumeSnapshot`の展開プロセスの一環として、Kubernetesチームは、コントロールプレーンに展開されるスナップショットコントローラーと、CSIドライバーと共に展開されるcsi-snapshotterと呼ばれるサイドカーヘルパーコンテナを提供します。`VolumeSnapshot`および`VolumeSnapshotContent`オブジェクトを管理し、`VolumeSnapshotContent`オブジェクトの作成と削除を担当します。サイドカーcsi-snapshotterは、`VolumeSnapshotContent`オブジェクトを監視し、CSIエンドポイントに対して`CreateSnapshot`および`DeleteSnapshot`操作をトリガーします。
```
[the remark this answers is not in the document]
Comment (KobayashiD27) [reply content/ja/docs/concepts/storage/volume-snapshots.md:29]: Thank you!

File: content/ja/docs/concepts/storage/volume-snapshots.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+クラスターにこの機能の機能があるかどうかを確認するには、次のコマンドを実行します。
+
+```yaml
+$ kubectl get crd volumesnapshotcontent -o yaml
+```
+
+ユーザーが既存の`VolumeSnapshot`から`PersistentVolumeClaim`を作成できるようにしたいが、ソースとは異なるボリュームモードを使用する場合は、`VolumeSnapshot`に対応する`VolumeSnapshotContent`にアノテーション`snapshot.storage.kubernetes.io/allowVolumeModeChange: "true"`を追加する必要があります。
+
+事前プロビジョニングされたスナップショットの場合、クラスター管理者が`Spec.SourceVolumeMode`を入力する必要があります。
```
Comment (atoato88) [inline content/ja/docs/concepts/storage/volume-snapshots.md:156]:
```suggestion
事前プロビジョニングされたスナップショットの場合、クラスター管理者が`spec.sourceVolumeMode`を入力する必要があります。
```
I created the PR(#38207) which update json path at here and merged now.
So please update here.

**Review by atoato88 — COMMENTED** [review]
I left just one comment.

**Review by atoato88 — CHANGES_REQUESTED** [review]
here is the comment.

**Comment by atoato88** [comment]
/lgtm
Thank you for updating!

**Comment by t-inu** [comment]
@nasa9084 Please enter `/approve` command again because it seems `approved` label has not added.

**Pull request merged**